NSrtm.Core/Pgm/GridGraph/PgmGridGraphInMemory.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using JetBrains.Annotations;
+
+namespace NSrtm.Core.Pgm.GridGraph
+{
+    public sealed class PgmGridGraphInMemory : IPgmGridGraph
+    {
+        private readonly UInt16[] _scaledUndulation;
+        private readonly PgmDataDescription _pgmParameters;
+
+        [CLSCompliantAttribute(false)]
```

Comment:
Maybe `GeoidUndulationGridData`? By word `provider` I was thinking about class providing data (like this one), not class providing instances. I would love this class name to put more emphasis on the fact that this is only provider of relatively raw data, but GeoidUndulationGrid is acceptable.